src/components/Books.js

```diff
@@ -1,74 +1,118 @@
-import React, { useState, useEffect } from 'react';
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import PropTypes from 'prop-types';
+import { fetchBooks, addBook, deleteBook } from '../redux/Books/booksSlice';
+import BookForm from './BookForm';
 
-const Books = () => {
-  const [books, setBooks] = useState([]);
+const Book = ({ objectFromBookList }) => {
+  const dispatch = useDispatch();
 
-  const storeForm = (form) => {
-    localStorage.setItem('form', JSON.stringify(form));
+  const handleDelete = async (id) => {
+    try {
+      await dispatch(deleteBook(id));
+      dispatch(fetchBooks());
+    } catch (error) {
+      // console.log(error);
+    }
```

Comment:
I have noticed you disabled eslint in this file it is against microverse policy to disable linters beause linters help us write clean code it would be nice to fix this by enabling eslint in all files where it has been disabled and kindly remove all console.log statements for gthe can cause errors for you do this in all files where console.log has been added and commented out code also makes ypur code base look untidy please kindly get rid of it

Reply:
done